apps/finance/app/src/components/Transfers.js

```diff
@@ -160,7 +197,7 @@ class Transfers extends React.Component {
                         key={transfer.transactionHash}
                         token={tokenDetails[toChecksumAddress(transfer.token)]}
                         transaction={transfer}
-                        wideMode={width > 800}
+                        wideMode={width > 768 + 219}
```

Comment:
Wondering if we should use a constant instead of `219` (is it the width of the side panel)?

Reply:
I did this on purpose so we would not miss it.